datahub-web-react/src/app/entity/dataset/DatasetEntity.tsx

```diff
@@ -142,7 +142,11 @@ export class DatasetEntity implements Entity<Dataset> {
                     component: QueriesTab,
                     display: {
                         visible: (_, _1) => true,
-                        enabled: (_, _2) => true,
+                        enabled: (_, dataset: GetDatasetQuery) => {
+                            const accessAspect = dataset?.dataset?.access;
+                            const rolesList = accessAspect?.roles;
+                            return !!accessAspect && !!rolesList && rolesList.length > 0;
+                        },
```

Comment:
shoot i think you added this to the wrong place! if we move this down to the `AccessManagement` tab below I think we're good to merge this guy

Reply:
now that is very weird, sorry!
It was correct in the [initial commit](https://github.com/datahub-project/datahub/pull/9610/commits/22e168bafbc29f8aff77028290b65b2236d1af78). Maybe broken due to a merge?

Updated it now.